src/target/stm32f1.c

```diff
@@ -440,6 +444,7 @@ static bool stm32f1_option_write(target *t, uint32_t addr, uint16_t value)
 	index = (addr - FLASH_OBP_RDP) / 2;
 	if ((index < 0) || (index > 7))
```

Comment:
the ()'s around the individual sub-clauses here are redundant and suppress warnings, please remove them